app/_components/commodities/components/GridBodyItem.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+
+import { formatThousands } from '@/app/_hooks/textFormatting';
+import { GridItem, SimpleGrid } from '@chakra-ui/react';
+import { ICommodityFormResponse } from '@/app/_types/commodity';
+import selectColor from '@/app/_hooks/fontColorSelector';
+import useColorMode from '@/app/_hooks/useColorMode';
+import { calculateTimeDifference, renderStationTypeIcon } from '../helpers';
+
+interface IGridBodyItemProps {
+  commodity: ICommodityFormResponse;
+  isBuying: boolean;
+}
+
+const GridBodyItem: React.FC<IGridBodyItemProps> = ({
+  commodity,
+  isBuying,
+}) => {
+  const { isDark } = useColorMode();
+
+  return (
+    <SimpleGrid
+      columns={[5, 7]}
+      width="100%"
+      gap={8}
+      fontSize="sm"
+      _odd={{
+        background: `${selectColor(isDark, '')}`,
```

Comment:
Can the alt rows be a tiny bit darker, it's barely visible on my monitor to the point where I had to squint to see it.

Reply:
fixed
![Screenshot 2023-08-29 at 16-30-12 EDPN - Commodities](https://github.com/ed-pilots-network/frontend/assets/67390534/cadbfa1c-389b-4445-927c-843e66170f38)
![Screenshot 2023-08-29 at 16-30-00 EDPN - Commodities](https://github.com/ed-pilots-network/frontend/assets/67390534/71d4e982-e4f5-47c5-bb53-0adf1b1cb50f)